include/gatekeeper_fib.h

```diff
@@ -33,6 +33,9 @@ enum gk_fib_action {
 	/* Forward the packet to the corresponding Grantor. */
 	GK_FWD_GRANTOR,
 
+	/* Given an IP prefix, only get the responsible Grantor. */
+	GK_GET_GRANTOR,
```

Comment:
I don't quite understand why this patch adds this functionality of `GK_GET_GRANTOR`, Is it necessary for the CPS block to be able to edit the GK table? If not, it should be in a patch of its own.

Reply:
Yes, it's necessary. Since we can only get the prefix and gateway information from CPS. For an ip prefix that needs to be forwarded to a grantor server. We first need to find the corresponding grantor server that is responsible for the prefix, which needs this functionality of ``GK_GET_GRANTOR``.